vault/core.go

```diff
@@ -3063,6 +3063,11 @@ func (c *Core) LogFormat() string {
 	return conf.(*server.Config).LogFormat
 }
 
+// LogLevel returns the log level provided by level provided by config, CLI flag, or env
+func (c *Core) LogLevel() string {
+	return c.logLevel
```

Comment:
This might be unlikely, but I am wondering if we should consider locking for this as well? I think recently you have made some endpoints available to modify this without the need to reload the config. So, this could potentially change at the time of reading the value which could cause a race condition.

Reply:
The recently added `sys/loggers` endpoints are used to modify `Core.allLoggers` which is a slice of loggers that a Vault core will track. I believe that this `logLevel` is read only. At least I couldn't find any cases where it was updated.